tests/codebuild/run_canarytest_china.sh

```diff
@@ -48,7 +57,11 @@ function download_installer_china(){
 }
 
 function create_eks_cluster() {
-  eksctl create cluster --name $CLUSTER_NAME --region $CLUSTER_REGION --auto-kubeconfig --timeout=30m --managed --node-type=c5.xlarge --nodes=1
+  eksctl_args=( --managed --nodes 1 --node-type=c5.xlarge --timeout=30m --region "$CLUSTER_REGION" --auto-kubeconfig )
```

Comment:
These configurations should not be hardcoded. If necessary, set as default with an option to override.